common/scripts/setup_env.sh

```diff
@@ -73,10 +73,10 @@ TIMEZONE=$(readlink $readlink_flags /etc/localtime | sed -e 's/^.*zoneinfo\///')
 export TIMEZONE
 
 export TARGET_OUT="${TARGET_OUT:-$(pwd)/out/${TARGET_OS}_${TARGET_ARCH}}"
-export TARGET_OUT_LINUX="${TARGET_OUT_LINUX:-$(pwd)/out/linux_amd64}"
+export TARGET_OUT_LINUX="${TARGET_OUT_LINUX:-$(pwd)/out/linux_${TARGET_ARCH}}"
```

Comment:
Note that changes in this file are made in the common-files repo and are propagated to the other repos.

See https://github.com/istio/common-files/blob/master/files/common/scripts/setup_env.sh#L76 for example.

Reply:
should i created i create another PR of that repo?
or just revert it

Reply:
You would create a PR on that repo with the change. Once that PR is merged, all the repos will be updated with the common file update. Then you can remove the file in this PR.